geanyctags/README

```diff
@@ -11,7 +11,9 @@ GeanyCtags adds a simple support for generating and querying ctags files for a G
 project. It requires that the ctags command is installed in a system path. On
 unix systems, distributions usually provide the ctags package; on Windows, the
 ctags binary can be found in the zip Windows distribution from the ctags home
-page (http://ctags.sourceforge.net).
+page (https://ctags.sourceforge.net). There is also a new, maintained implementation
```

Comment:
Given the last release of ectags was 2009 it seems better to not mention it. Uctags has 15 years of improvement and makes Windows and mac binaries.

Reply:
Good point, I've updated this section. I kept the information about ctags though so people know what the difference between ctags and universal ctags is (I think distributions still package the unmaintained ctags).